pkg/adaptor/hypervisor/aws/service.go

```diff
@@ -173,6 +173,20 @@ func (s *hypervisorService) StartVM(ctx context.Context, req *pb.StartVMRequest)
 		},
 	}
 
+	if authJSON, err := os.ReadFile(cloudinit.DefaultAuthfileSrcPath); err == nil {
```

Comment:
nit: Should this be moved into a single file? OR is the plan to add this into the refactor steps being done in a different PR.